l2tdevtools/build_helpers/dpkg.py

```diff
@@ -369,6 +402,39 @@ def _RemoveOlderSourceDPKGPackages(self, project_name, project_version):
         logging.info('Removing: {0:s}'.format(filename))
         os.remove(filename)
 
+  def _RunLSBReleaseCommand(self, option='-a'):
+    """Runs the lsb-release command (/usr/bin/lsb_release).
+
+    This functions ignores comment lines, lines that are not UTF-8 formatted
```

Comment:
This comment doesn't seem to be valid - please remove

Reply:
done